### PR TITLE
Added environment specific configs files (just a greeting message right

### PR DESCRIPTION
**Pull request by anoveskey**
now)

### CODE REVIEW

File: app/config.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+export const getConfigs = () => {
+  switch (process.env.REACT_APP_ENVIRONMENT) {
```
Comment (jmiller-nexient) [inline app/config.ts:2]: We won't have multiple files.  So no `switch` statement. Only a single require statement returning the one (and only one) config file (named 'config.json').

File: public/config.dev.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (jmiller-nexient) [inline public/config.dev.json:1]: Rename this file 'config.json'.

File: public/config.local.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (jmiller-nexient) [inline public/config.local.json:1]: Delete this file.

File: public/config.prod.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (jmiller-nexient) [inline public/config.prod.json:1]: Delete this file.

File: public/config.qa.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (jmiller-nexient) [inline public/config.qa.json:1]: Delete this file.

File: public/config.stage.json [hunk 6]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (jmiller-nexient) [inline public/config.stage.json:1]: Delete this file.

**Comment by anoveskey** [comment]
I saw the comment on commit message formatting a bit late.

**Comment by jmiller-nexient** [comment]
> I saw the comment on commit message formatting a bit late.

No worries. Best we get the kinks worked out here before we are pushing to DTE's GitHub repo.

**Pull request merged**